packages/aws-cdk-lib/aws-events/lib/event-bus.ts

```diff
@@ -80,6 +80,17 @@ export interface EventBusProps {
    */
   readonly eventSourceName?: string;
 
+  /**
+   * The event bus description.
+   *
+   * The description must be maximum 512 characters long.
```

Comment:
This phrasing feels a little confusing to me. Can we change it to something more like:
```suggestion
   * The description can be up to 512 characters long.
```